### DISCUSSION TITLE
FastNoiseLite Extension

### DISCUSSION THREAD

**Discussion by Vxerz** [TurboWarp/extensions#1814] [Show and tell]
https://github.com/Vxerz/FastNoiseLiteScratch

This is an extension i made that adds Auburn's FastNoiseLite project (https://github.com/Auburn/FastNoiseLite) as an extension, there are 2 blocks, create noise and get noise. You can have as many noises as you want, and there are many parameters for the noise.
![noiseBlocks](https://github.com/user-attachments/assets/853b3153-954c-4ee7-ba15-99410c88cc30)
![image](https://github.com/user-attachments/assets/9d6c419f-d4bc-438e-a7a7-c1a3e785dc0f)![image](https://github.com/user-attachments/assets/12dad5a7-96a4-4cc1-ada9-af443a052f89)![image](https://github.com/user-attachments/assets/5ac6fdcf-9bea-4c8d-99bf-d768d1db958d)![image](https://github.com/user-attachments/assets/f7d179e2-db72-4cb3-9430-aa1ccea6fdc2)

Thought this would be nice to have on Turbowarp since there aren't any unsandboxed extensions for noise yet.

I don't need any credit since I'm not the one who made the noise, and if someone wants to add more features to it all of the code is in the link at the top so feel free to do so.

**Comment by CubesterYT** [comment]
You should add it to the extension gallery yourself. Open a Pull Request!

**Reply by Vxerz** [reply]
i aint really use github at all so im not too sure how to do that tbh

**Reply by CubesterYT** [reply]
Fork the extensions repository
Make a new branch on your fork (name it whatever)
Make a folder with your name among the other named folders
Upload the extension there
Modify the .json file that lists all the extensions
Submit a PR!

**Reply by CubesterYT** [reply]
These are very scuffed instructions, but they work.

**Reply by Vxerz** [reply]
ah ok that makes sense

**Reply by Vxerz** [reply]
thank you!

**Reply by CubesterYT** [reply]
You're welcome!

**Reply by yuri-kiss** [reply]
more to review :dance:
